internal/cmd/users.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edulinq/autograder/internal/api/core"
+)
+
+const INDENT = "    "
+
+var rows [][]string
+
+func ListUsers(users []*core.ServerUserInfo, courseUsers bool, table bool) {
```

Comment:
`courseUsers` is unused.

I know you want it included because it will eventually be needed, but it makes more sense for this PR to leave it out. 

We can easily expand the function and update the one function call when we have a need for listing course users (soon hopefully).